modules/virt-configuring-a-live-migration-policy.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * virt/live_migration/virt-configuring-live-migration-policies.adoc
+
+
+:_content-type: PROCEDURE
+[id="virt-configuring-a-live-migration-policy_{context}"]
+= Configuring a live migration policy
+
+Use the `MigrationPolicy` custom resource definition (CRD) to define migration policies for one or more groups of selected virtual machines (VMs).
+
+You can specify groups of VMs by using any combination of the following:
+* VM labels such as `size`, `os`, and `gpu`.
```

Comment:
Maybe it's valuable to emphasize here that every label is valid to use and `size`, `os`, etc are just examples